### ISSUE TITLE
[BUG]: Contrast violations not reported when overflow:hidden is set

### ISSUE THREAD

**Issue by wkeese** [IBMa/equal-access#2110]
### Project

accessibility-checker-engine

### Browser

Chrome

### Operating system

MacOS

### Description

Accessibility checker doesn't flag text for contrast ratio violations if the node has overflow:hidden.

### Steps to reproduce

Save this HTML to disk and open in Chrome, then run accessibility checker:

```html
<html lang="en">
<head>
    <meta charset="utf-8">
    <title>contrast test</title>
    <style>
        div {
            background: #eee;
        }
        span {
            overflow: hidden;
            color: #fff;
        }
    </style>
</head>
<body>
<main>
    <div>
        <span>Customers</span>
    </div>
</main>
</body>
</html>
```

Accessibility checker should flag the "Customers" text as having a low contrast ratio, but it doesn't.

If I remove the `overflow: hidden` then it flags the error.

Note that the text is fully visible on the screen regardless of the `overflow` setting.
